sympy/vector/tests/test_coordsysrect.py

```diff
@@ -390,3 +390,24 @@ def test_rotation_trans_equations():
     assert a._rotation_trans_equations() == (a.x, a.y, a.z)
     b = a.orient_new_axis('b', 0, -a.k)
     assert b._rotation_trans_equations() == (b.x, b.y, b.z)
+
+
+def test_transformation_composition():
+    a = CoordSys3D('a')
+    eq1 = a._rotation_trans_equations()
+    eq2 = a._transformation_equations()
+    assert a._transformation_composition(eq1, eq2) == (a.x, a.y, a.z)
+    from sympy import symbols
+    q0 = symbols('q0')
+    b = a.orient_new_axis('b', q0, a.k)
+    eq1 = b._transformation_equations()
+    eq2 = b._rotation_trans_equations()
+    assert b._transformation_composition(eq1, eq2) ==\
```

Comment:
I can remove these example if you want:). But for debugging it's quite convenient to keep this because when results will be irrational I will easily see were something goes wrong.